File: front-end/src/app/dto/learning/learning.res.dto.ts
import { MaterialResDto } from '@dto/material/material.res.dto';
import { TaskResDto } from '@dto/task/task.res.dto';

export interface LearningResDto {
  id: number;
  learningName: string;
  learningCode: string;
  startTime: string;
  endTime: string;
  materials: MaterialResDto[];
  tasks: TaskResDto[];
}

File: back-end/src/main/java/com/ocire/lms/dto/learning/LearningResDto.java
package com.ocire.lms.dto.learning;

public class LearningResDto {
	private Long id;
	private String learningName;
	private String learningCode;
	private String startTime;
	private String endTime;
	
	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public String getLearningName() {
		return learningName;
	}

	public void setLearningName(String learningName) {
		this.learningName = learningName;
	}

	public String getLearningCode() {
		return learningCode;
	}

	public void setLearningCode(String learningCode) {
		this.learningCode = learningCode;
	}

	public String getStartTime() {
		return startTime;
	}

	public void setStartTime(String startTime) {
		this.startTime = startTime;
	}

	public String getEndTime() {
		return endTime;
	}

	public void setEndTime(String endTime) {
		this.endTime = endTime;
	}

}
